tools/ui-components/src/alert/alert.tsx

```diff
@@ -32,22 +32,17 @@ export function Alert({
   const variantClass = variantClasses[variant];
 
   const classes = [
-    'relative p-4 mb-6 border border-transparent break-words',
+    'flex items-start p-4 mb-6 border border-transparent break-words',
     variantClass,
-    isDismissable ? 'pr-10' : '',
     className
   ].join(' ');
 
   return (
     <div className={classes} role='alert'>
+      <div className='grow'>{children}</div>
       {isDismissable && (
-        <CloseButton
-          className='absolute right-4'
-          label={dismissLabel}
-          onClick={onDismiss}
-        />
+        <CloseButton label={dismissLabel} onClick={onDismiss} />
```

Comment:
Apart from swapping the DOM order, I took the opportunity to change the styling approach a bit.

Currently we are positioning the close button using `absolute` and spacing classes (`right-4` and `pr-10`). Personally I would avoid playing with spacing values when I can, just because I think the layout would be easier to break. 

I changed the implementation a bit and use Flex instead. The wrapper div is a flex container with its items (the message and the button) aligned at the start of the cross axis (via the `flex` and `items-start` classes). The message is wrapped in a div that has `flex-grow` set to `1` (via the `grow` class) in order for it to stretch and fill the container.

There should be no regressions or visual changes as a result of this.